packages/manager/modules/pci/src/projects/project/storages/containers/container.class.js

```diff
@@ -55,4 +58,9 @@ export default class Container {
   getObjectById(objectId) {
     return find(this.objects, { name: objectId });
   }
+
+  isEncrypted() {
+    const { sseAlgorithm } = this.encryption;
+    return sseAlgorithm && sseAlgorithm !== NO_ENCRYPTION_VALUE;
+  }
```

Comment:
Have you ever considered using a `get` for this kind of property ?
```suggestion
  get isEncrypted() {
    const { sseAlgorithm } = this.encryption;
    return sseAlgorithm && sseAlgorithm !== NO_ENCRYPTION_VALUE;
  }
```
Then in your code / template
```
this.isEncrypted
$ctrl.isEncrypted
```

Reply:
fixed !